pvlib/iotools/psm3.py

```diff
@@ -40,6 +39,21 @@
     'Precipitable Water': 'precipitable_water',
 }
 
+# Dictionary mapping pvlib names to PSM3 request names
+# Note, PSM3 uses different names for the same variables in the
+# response and the request
+REQUEST_VARIABLE_MAP = {
```

Comment:
At first I wondered why `ghi` and its friends are not included here.  I guess it's because the keys and values are identical, so no need to include them?  Maybe a good idea to include them anyway for explicitness.